.github/workflows/mobile-maintenance.yml

```
@@ -0,0 +1,255 @@
+      issues: write
+    
+    steps:
+      - name: 🏗 Setup repo
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: 🏗 Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+<<<<<<< HEAD
```

actionlint (1.7.10)

could not parse as YAML: could not find expected ':'

(syntax-check)